app/src/main/java/com/example/unsplashphotos/ui/photofullscreen/PhotoFullViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.unsplashphotos.ui.photofullscreen
+
+
+import androidx.lifecycle.ViewModel
+import com.example.unsplashphotos.data.model.Photo
+import com.example.unsplashphotos.domain.usecase.PhotoFullScreenUseCase
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import timber.log.Timber
+import javax.inject.Inject
+
+@HiltViewModel
+class PhotoFullViewModel @Inject constructor(private val photoFullScreenUseCase: PhotoFullScreenUseCase) :
+    ViewModel() {
+
+    lateinit var flow: Flow<Photo?>
```

Comment:
use MutableStateFlow and expose only stateflow. See the difference between a flow and MutableStateFlow